specs/test_formats/ssz_generic/README.md

```diff
@@ -16,5 +16,11 @@ Test format documentation can be found here: [uint test format](./uint.md).
 *Note*: The current Phase 0 spec does not use larger uints, and uses byte vectors (fixed length) instead to represent roots etc.
 The exact uint lengths to support may be redefined in the future.
 
-Extension of the SSZ tests collection is planned, with an update to the new spec-maintained `minimal_ssz.py`;
- see CI/testing issues for progress tracking.
+## Recommendation
+
+For SSZ testing directly applicable to test-networks, refer to the [`ssz_static`](../ssz_static/README.md) tests,
+ which cover SSZ behavior for phase-0, for all container types.
```

Comment:
Now sure about all these new lines. IMO you can just put it with the previous line.

Reply:
see 795a778